library/fedora

```diff
@@ -12,6 +12,12 @@ arm32v7-Directory: armhfp/
 arm64v8-Directory: aarch64/
 ppc64le-Directory: ppc64le/
 
+Tags: 26-modular
+Architectures: amd64
+GitFetch: refs/heads/26
```

Comment:
@tianon it should be, yes ... I apparently can't type and chew bubblegum at the same time. :)